beacon_node/src/cli.rs

```diff
@@ -282,7 +291,20 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                        for a beacon node being referenced by validator client using the --proposer-node flag. This configuration is for enabling more secure setups.")
                 .takes_value(false),
         )
-
+        .arg(
+            Arg::with_name("inbound-rate-limiter")
+            .long("inbound-rate-limiter")
+            .help(
+                "Enables the inbound rate limiter (requests received by this node).\
+                \
+                Rate limit quotas per protocol can be set in the form of \
+                <protocol_name>:<tokens>/<time_in_seconds>. To set quotas for multiple protocols, \
+                separate them by ';'. If the inbound rate limiter is enabled and a protocol is not \
+                present in the configuration, the default quotas will be used."
+            )
+            .min_values(0)
+            .hidden(true)
+        )
```

Comment:
A suggestion that removes the need for two flags https://github.com/sigp/lighthouse/commit/488011941899842569146e0589b848c99646f58e

In summary, the option will be a "normal" one that requires a value. The value is either "disabled" or a configuration string. This way we don't need to add a second flag for disabling